schema.graphql

```diff
@@ -37,11 +37,17 @@ type Address implements Node {
     """Returns the elements in the list that come before the specified cursor"""
     before: String
 
+    """If supplied, filters by the specific event"""
+    event: String
+
     """Returns the first n elements from the list."""
     first: Int
 
     """Returns the last n elements from the list."""
     last: Int
+
+    """If supplied, filters by the specific ticket type"""
+    ticketTypeId: String
```

Comment:
can we query with this parameter as well?

Reply:
yes this is the change for adding the filter parameter to the typegen file. I didn't regenerate this file in my last PR where I actually added the params to the query.